### ISSUE TITLE
TypeError: t.loginPopup$ is undefined

### ISSUE THREAD

**Issue by mirekys** [oarepo/cesnet-demo-ui#12]
On FF browsers `loginPopup$` is accessed before it is registered on Vue app instance.
This causes following errors:

- For anonymous user:
```
TypeError: t.loginPopup$ is undefined
in
{key:"performLogIn",value:async function(t){return this.setState(w.LOADING),await t.loginPopup$.waitForLogin(),this.loggedLocally}}
```

- For authenticated user:

```
Uncaught (in promise) TypeError: Cannot read property 'waitForLogin' of undefined
    at Object.value (vendor.0bead706.js:34)
    at v.u (vendor.0bead706.js:39)
    at Array.<anonymous> (vendor.0bead706.js:18)
    at v.dispatch (vendor.0bead706.js:18)
    at v.dispatch (vendor.0bead706.js:18)
    at Object.r.dispatch (vendor.0bead706.js:18)
    at Object.performLogIn (vendor.0bead706.js:39)
    at Object.value (vendor.0bead706.js:34)
```

**Comment by mirekys** [comment]
This error is caused by: [vue-popup-login#116](https://github.com/oarepo/vue-popup-login/issues/116)

**Comment by mirekys** [comment]
Fixed by [vue-popup-login#118](https://github.com/oarepo/vue-popup-login/pull/118)
